samples/start-app/php-symfony/src/Controller/AppActivationCallbackController.php

```diff
@@ -16,23 +19,75 @@ final class AppActivationCallbackController extends AbstractController
     public function __construct(
         private readonly AppActivationCallback    $callback,
         private readonly TokenRepositoryInterface $tokenRepository,
-        private readonly string                   $projectDir)
+        private readonly UserRepositoryInterface  $userRepository,
+        private readonly LoggerInterface          $logger,
+        private readonly string                   $projectDir,
+        private readonly string                   $openIdAuthentication)
     {
     }
 
     #[Route('/callback', name: 'callback', methods: ['GET'])]
     public function __invoke(): Response
     {
         session_start();
-        $this->callback->execute($_SESSION, $_GET['state'], $_GET['code']);
 
-        if (!$this->tokenRepository->hasToken()) {
-            return new Response(file_get_contents($this->projectDir . '/templates/no_access_token.html'));
-        }
+        try{
+            $data = $this->callback->execute($_SESSION, $_GET['state'], $_GET['code']);
+
+            $response = new Response();
+
+            $divToReplace = "<div>UserInformation</div>";
+            $divUserInformation = "";
+
+            if (!$this->tokenRepository->hasToken()) {
+                return $response->setContent(
+                    str_replace(
```

Comment:
I think you dont need to replace here, juste delete `<div>UserInformation</div>` in template

Reply:
I would not do that as the error template could be used in another use case, where cookies are correctly filled and so the user would be connected.

Reply:
Done